Serilog.sln

```diff
@@ -1,25 +1,12 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
```

Comment:
With all those magic guids...yeah I can see that this is not handy. Will do a change in VS instead of Xamarin.